wyvern/wyvern_request.py

```diff
@@ -77,6 +77,3 @@ def parse_fastapi_request(
             feature_map=FeatureMap(feature_map={}),
             request_id=request_id,
         )
-
-    def extend_feature_map(self, feature_map: FeatureMap) -> None:
-        self.feature_map.feature_map.update(feature_map.feature_map)
```

Comment:
this is a bug. the feature_map key is "identifier". if a real-time feature identifier is the same as a feature store identifier, the realtime features for this identifier is going to override the the batch features of the same identifier.

The expected behavior should be a merge but this is the what the code does.